client/src/components/AddClimbedBoulderForm.jsx

```diff
@@ -0,0 +1,184 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const AddClimbedBoulderForm = ({saveClimbedBoulder}) => {
+  const [projected, setProjected] = useState(false);
+  const [attempts, setAttempts] = useState("");
+  const [result, setResult] = useState();
+  const [liked, setLiked] = useState(false);
+  const [levelFeedback, setLevelFeedback] = useState();
+  
+
+  return (
+    <BoulderForm
+      onSubmit={(event) => {event.preventDefault();
+        saveClimbedBoulder(projected, attempts, result, liked, levelFeedback);
+      }}
+    >
+      <p>Save your achievement!</p>
+      <div>
+        <label htmlFor="projected">Projected: </label>
+        <input
+          type="checkbox"
+          id="projected"
+          name="projected"
+          value={projected}
+          onChange={event => {
+            setProjected(event.target.checked);
+          }}
+        ></input>
+      </div>
+      <div>
+        <label htmlFor="attempts">Attempts: </label>
+        <input
+          type="number"
+          id="attempts"
+          name="attempts"
+          value={attempts}
+          onChange={event => {
+            setAttempts(event.target.value);
+          }}
+        ></input>
+      </div>
+      <fieldset>
+        <legend>Your result:</legend>
+        <RadioButtonWrapper>
+           <div>
+          <input
```

Comment:
Prettier does not seem to work here.

Reply:
It works but I have to save it manually. I need to check why...